docs/tutorials/control/scuttle-gamepad.md

```diff
@@ -60,25 +63,47 @@ In the **Create component** panel, configure a [gamepad](/components/input-contr
 
 ![Blank configuration JSON](/tutorials/scuttle-gamepad/gamepad-config.png)
 
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+If instead of using the config builder, you prefer to write raw JSON, switch to **Raw JSON** mode on the **Config** tab.
+Inside the `"components": []` section of your config, add the following configuration for your [gamepad](/components/input-controller/gamepad/):
```

Comment:
```suggestion
If instead of using the config builder, you prefer to write raw JSON, switch to [**Raw JSON** mode](/manage/configuration/#the-config-tab) on the **Config** tab.
Inside the `components` array of your config, add the following configuration for your [gamepad](/components/input-controller/gamepad/):
```